docs/dg/devOpsGuide.md

```diff
@@ -28,9 +28,9 @@ This page documents the various components that form part of the DevOps infrastr
 
 All three types of tests in the test suite are run in a single GitHub Actions workflow called "Continuous Integration". The steps are defined in [`integration.yml`](https://github.com/reposense/RepoSense/blob/master/.github/workflows/integration.yml) and are split into four types of jobs:
```

Comment:
Given the comment below, I think the 'three' might need to change if you change below. It should probably be 4 at this point anyway.

Reply:
The three here refers to the unit tests, system tests and frontend tests, rather than the OS